src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderAlert.kt

```diff
@@ -45,9 +65,60 @@ class OffenderAlertBuilderRepository(
   private val alertCodeRepository: ReferenceCodeRepository<AlertCode>,
   private val alertTypeRepository: ReferenceCodeRepository<AlertType>,
   private val workFlowActionRepository: ReferenceCodeRepository<WorkFlowAction>,
+  private val jdbcTemplate: NamedParameterJdbcTemplate,
 ) {
   fun save(alert: OffenderAlert): OffenderAlert =
-    repository.save(alert)
+    repository.saveAndFlush(alert)
+
+  fun updateAudit(
+    id: OffenderAlertId,
+    createDatetime: LocalDateTime,
+    createUsername: String,
+    modifyUserId: String?,
+    modifyDatetime: LocalDateTime?,
+    auditTimestamp: LocalDateTime?,
+    auditUserId: String?,
+    auditModuleName: String?,
+    auditClientUserId: String?,
+    auditClientIpAddress: String?,
+    auditClientWorkstationName: String?,
+    auditAdditionalInfo: String?,
+  ) {
+    jdbcTemplate.update(
```

Comment:
chosen to do all this by SQL since these fields are set by NOMIS so feels a shame to corrupt the r/w JPA mappings. And also I'm not convinced we will need this code in the future